src/elements/select/select.svelte

```diff
@@ -25,6 +25,9 @@ export let tooltip = '';
 export let state: 'info' | 'warn' | 'error' | '' = 'info';
 export let showpill = 'false';
 export let clearable = 'true';
+export let button = 'false';
```

Comment:
Can we change this name to something like "withbutton" or something? Just feels more obvious that it's a boolean value than "button"

Reply:
yup